frontend/server/src/Controllers/Course.php

```diff
@@ -39,7 +39,7 @@
  * @psalm-type CourseNewPayload=array{is_curator: bool, is_admin: bool}
  * @psalm-type CourseEditPayload=array{admins: list<CourseAdmin>, assignmentProblems: list<ProblemsetProblem>, course: CourseDetails, groupsAdmins: list<CourseGroupAdmin>, identityRequests: list<IdentityRequest>, selectedAssignment: CourseAssignment|null, students: list<CourseStudent>, tags: list<string>}
  * @psalm-type StudentProgressPayload=array{course: CourseDetails, students: list<StudentProgress>, student: string}
- * @psalm-type StudentsProgressPayload=array{course: CourseDetails, students: list<StudentProgress>}
+ * @psalm-type StudentsProgressPayload=array{course: CourseDetails, problems: array<string, string>, students: list<StudentProgress>}
```

Comment:
```suggestion
 * @psalm-type StudentsProgressPayload=array{course: CourseDetails, problemTitles: array<string, string>, students: list<StudentProgress>}
```

esto no contiene los problemas, sino el mapping de títulos por alias. igual en el resto del cambio.

Reply:
Listo